### PR TITLE
Switch numbers to const

### PR DESCRIPTION
**Pull request by cbonaudo**
Switched the solution for numbers to const

### CODE REVIEW

**Comment by AryanJ-NYC** [comment]
Hey @cbonaudo. All "replacing `var` to `const`" pull requests should have a base branch of `es6`, not `master`.

Reference #229.

**Comment by AnshulMalik** [comment]
@cbonaudo There is an extra commit for `number-to-string` can we just have the second commit?

File: problems/number-to-string/problem_zh-tw.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 這時，你可以使用 `.toString()` 方法。例如：
 
 ```js
-var n = 256;
+const n = 256;
```
Comment (AryanJ-NYC) [inline problems/number-to-string/problem_zh-tw.md]: This should be `let`, not `const`, as it is reassigned.

**Comment by ledsun** [comment]
@cbonaudo Are you planning to renew?

**Comment by ledsun** [comment]
Close to clarify the status of Pull requests. Please reopen if you want to continue development.

**Pull request closed without merging**